frontend/tests/ui/test_contest.py

```diff
@@ -116,13 +125,31 @@ def test_user_ranking_contest(driver):
                 (By.CSS_SELECTOR, '#ranking')))
 
         run_accepted_user = driver.browser.find_element_by_xpath(
-            '//td[contains(@class, "accepted")]/preceding-sibling::td[1]')
+            '//td[contains(@class, "accepted")]/preceding-sibling::td[@class='
+            '"user"]')
         assert run_accepted_user.text == user1, run_accepted_user
 
         run_wrong_user = driver.browser.find_element_by_xpath(
-            '//td[contains(@class, "wrong")]/preceding-sibling::td[1]')
+            '//td[contains(@class, "wrong")]/preceding-sibling::td[@class='
+            '"user"]')
         assert run_wrong_user.text == user2, run_wrong_user
 
+        contestants_full_list = driver.browser.find_elements_by_xpath(
+            '//*[@id="ranking"]/div/table/tbody/tr/td[@class="user"]')
+        assert len(contestants_full_list) == 4, contestants_full_list
+        full_list = [user1, user2, user3, driver.user_username]
+        find_users_in_list(contestants_full_list, full_list)
+
+        driver.wait.until(
+            EC.element_to_be_clickable(
+                (By.XPATH, '//input[@class = "toggle-contestants"]'))).click()
+
+        invited_contestant_list = driver.browser.find_elements_by_xpath(
+            '//*[@id="ranking"]/div/table/tbody/tr/td[@class="user"]')
+        assert len(invited_contestant_list) == 3, invited_contestant_list
+        invited_list = [user1, user2, driver.user_username]
+        find_users_in_list(invited_contestant_list, invited_list)
```

Comment:
qué te parece mejor usar [`set`](https://docs.python.org/3/tutorial/datastructures.html#sets)? así no necesitas hacer esta función auxiliar que corre en O(n²):

```suggestion
        expected_invited_usernames = {user1, user1, driver.user_username}
        actual_invited_usernames = {element.text for element in invited_contestant_list}
        assert expected_invited_usernames == actual_invited_usernames, (...)
```

así de paso estás checando que los conjuntos sean iguales en vez de checar si uno es subconjunto del otro, lo cual es un poco más robusto.

igual arriba.